scripts/update-json-schema.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Ensures that we run on Travis
+if [ "$TRAVIS_BRANCH" != "master" ] || [ "$GENERATE_JSON_SCHEMA" != "yes" ] || [ "$TRAVIS_SECURE_ENV_VARS" == "false" ] || [ "$TRAVIS_PULL_REQUEST" != "false" ] ; then
+    echo "Must be: a merged PR on the master branch, GENERATE_JSON_SCHEMA=yes, TRAVIS_SECURE_ENV_VARS=false"
+    exit 0
+fi
+
+KEDGE_REPO_NAME="kedge-json-schema"
+GENERATOR_REPO="git@github.com:kedgeproject/json-schema.git"
+DEPLOY_KEY="scripts/deploy_key"
+KEDGE_JSON_SCHEMA_IMAGE="containscafeine/kedge-json-schema:latest"
+GIT_USER="kedge-bot"
+GIT_EMAIL="shubham@linux.com"
+OUTPUT_DIR="master"
+
+# decrypt the private key
+openssl aes-256-cbc -K $encrypted_91569b511922_key -iv $encrypted_91569b511922_iv -in "$DEPLOY_KEY.enc" -out "$DEPLOY_KEY" -d
+chmod 600 "$DEPLOY_KEY"
+eval `ssh-agent -s`
+ssh-add "$DEPLOY_KEY"
+
+# clone the JSON Schema Generator repo
+git clone "$GENERATOR_REPO" "$KEDGE_REPO_NAME"
+mkdir -p "$KEDGE_REPO_NAME"/"$OUTPUT_DIR"
+cd "$KEDGE_REPO_NAME"/"$OUTPUT_DIR"
+docker run --rm -v `pwd`:/data:Z "$KEDGE_JSON_SCHEMA_IMAGE"
+
+# add relevant user information
+git config user.name "$GIT_USER"
+
+# email assigned
+git config user.email "$GIT_EMAIL"
+git add --all
```

Comment:
I don't think it is safe to have `--all` here. It should be explicit and add just files with json schema

Reply:
Agreed.

Reply:
@kadel @pradeepto this is happening inside the kedge-json-schema/master directory, and everything generated there is an output of json-schema-generator. The output files vary so it's difficult to keep track of which exact files, but since this is happening inside the repo, I think it's safe to add commit all of the files.

Reply:
all generated files are in `schema` directory, is that right? If not then we should have all generated files in some directory and not spread them across the repo. Than we can add just that directory

Reply:
@kadel ack, changed this to only add the `master/` directory in JSON Schema repo where the output will be spit to.